src/data/customer/Order.js

```diff
@@ -20,17 +20,20 @@ import {
 
 /**
  * Get template for order upload
+ * @param {object} pickupDate # {pickupDate (mandatory)}
```

Comment:
@pncsantos are you sure the pickupDate is mandatory? I don't think it is.

Reply:
@maartenhemmes 
I updated the javascript docs and also rebased into the dev branch.